src/public/CowSwap.json

```diff
@@ -436,6 +436,22 @@
       "decimals": 18,
       "chainId": 1,
       "logoURI": "https://gateway.pinata.cloud/ipfs/QmSxuQpx3UadQY4B2i5AQAaU97JZqkhcm9dCZXXNsNziYR"
+    },
+    {
+      "symbol": "icETH",
+      "name": "Interest Compounding ETH Index",
+      "address": "0x7C07F7aBe10CE8e33DC6C5aD68FE033085256A84",
+      "decimals": 18,
+      "chainId": 1,
+      "logoURI": "https://assets.website-files.com/60d237b6ea65be721ff4bbfa/62388a82c46d9a1dfb3dc506_icETH-token-logo.png"
```

Comment:
nit: All of our logos are uploaded to IPFS (which ensures availability even if they are no longer accessible at the specified URL). I think we should do the same for this here.

Reply:
I thought so too (mentioned in the PR description) however it's not true that all other token images are hosted on IPFS (a lot of them are hosted via the 1Inch list). We could also host them in this repo. 

I can take the time to convert those and also write a proper readme if we agree on a process.

Reply:
Ah, sorry - I just looked at the ones immediately above the added tokens.